apps/session-manager/src/services/redis-session-storage.ts

```diff
@@ -734,6 +735,71 @@ export const set =
     );
   };
 
+/**
+ * Delete all tokens related to the user
+ * @param user
+ * @returns
+ */
+export const deleteUser: (
+  user: User,
+) => (deps: RedisRepo.RedisRepositoryDeps) => TE.TaskEither<Error, boolean> =
+  (user) => (deps) => {
+    const tokens: ReadonlyArray<string> = pipe(
+      getUserTokens(user),
+      R.collect(S.Ord)((_, { prefix, value }) => `${prefix}${value}`),
+    );
+
+    return pipe(
+      tokens,
```

Comment:
We can use only one pipe, but is not needed.

```suggestion
  (user) => (deps) =>
    pipe(
      getUserTokens(user),
      R.collect(S.Ord)((_, { prefix, value }) => `${prefix}${value}`),
      (tokens) =>
        pipe(
          tokens,
```